docs/digital/exports/main-export-fields.md

```diff
@@ -133,19 +133,275 @@ Other custom data can be present in the thread export depending on the source ty
 | entry_id             | Reply Assistant Entry ID                                  | ObjectId | 5a4bbf3713047d6d123061d0                           | [Sensitive](../#sensitive-columns)                                   |
 | version_id           | Reply Assistant Version ID                                | ObjectId | 5a4bbf3713047d6d123061d0                           | [Sensitive](../#sensitive-columns)                                   |
 
-List of events containing a `task_id`:

```

Comment:
you should not remove this part

Reply:
I had a discussion with @kalilz4485 about this, he was of the opinion that the list of these events is already included in the table, so we can remove them.

Reply:
the table we're adding is listing all the possible events whereas this one was listing the events that have a `task_id` in it, it may be okay to remove it but if you do please also update the task_id row's comment

Reply:
Done

Reply:
yep we looked at the blame and it was added arbitrarily by someone, with the list its obvious all task.something events have a task.id (previous list didnt have event name just description as well) so IMO this part was useless